ubuntudesign/landscape.canonical.com
# Updating the cloud vanilla dependancy to 0.0.18

# Done

Updates the cloud vanilla dependancy to version 0.0.18

# QA

Pull down the branch and run ```npm install``` and ```gulp build``` to create the styles and then check any of the static pages
